### ISSUE TITLE
Does not work with Julia 1.4

### ISSUE THREAD

**Issue by sjoelund** [OpenModelica/OMJulia.jl#36]
The following works fine in Julia 1.1, but 1.4 refuses to start:
```
import OMJulia
om=OMJulia.OMCSession()
OMJulia.sendExpression(om, "1+1")
```

**Comment by arun3688** [comment]
@sjoelund  I tested with julia 1.4.2 , it is working

**Comment by sjoelund** [comment]
1.4.2 hangs for me as well. I get:
```
WARNING: Compat.Sys is deprecated, use Base.Sys instead.
  likely near REPL[2]:1
OMJulia.OMCSession(...)
```
And 
```
martin   15722  0.0  0.0 319768 20252 pts/5    Sl+  14:56   0:00 omc --interactive=zmq +z=julia.5Q6UlM8rg7
```

But then nothing happens. The log-file just says:
```
tcp://127.0.0.1:43427
```

So I guess there is something not working with ZeroMQ

**Comment by arun3688** [comment]
@sjoelund  I am not even getting such warnings in windows.

![image](https://user-images.githubusercontent.com/6870949/86009518-9c27ee00-ba1a-11ea-8320-18022dea09eb.png)

i updated all  the packages and tested it.

**Comment by sjoelund** [comment]
I also updated my packages and tested with 1.4.2 again :)

**Comment by sjoelund** [comment]
If I update to OMJulia#master I also don't get that warning. But it still hangs...

**Comment by arun3688** [comment]
@sjoelund , I was able to reproduce the problem in linux,  i found the problem related to hanging, It is the "Parser" module which takes too long for the first time compilation, and not the ZMQ,  the ZMQ is working fine . I did some trials in the dev server and found the following 

### Without disabling Parser module: 

```
>> using OMJulia
>> omc= OMJulia.OMCSession()
>> sendExpression(omc, "1+1")  // waited for 10 minutes approximately, got the results back , Sometimes it takes longer than 10 minutes for the first time compliation
```

### After disabling Parser module: 
 #return Parser.parseOM(unsafe_string(message))

```
>> using OMJulia
>> omc= OMJulia.OMCSession()
>> sendExpression(omc, "1+1")  //  not hanging, got the results immediately 
```
For some reasons this is  not happening in windows,  You can try this and see, It is the parser module which causes the hanging

**Comment by sjoelund** [comment]
https://gist.github.com/sjoelund/af90649890dec720cb9d7aab58c3a196

```
[martin@mega src]$ time julia-1.4.2 -O0 test.jl 
Any[2]

real    0m1,148s
user    0m1,155s
sys     0m0,367s
[martin@mega src]$ time julia-1.4.2 -O1 test.jl 
Any[2]

real    0m37,918s
user    0m37,624s
sys     0m0,475s

$ time julia-1.4.2 test.jl  
Any[2]

real    12m6,189s
user    11m53,341s
sys     0m1,267s
```

What are the times on that file in Windows? (Running from command-line)
If it's very different from Linux I guess we should make a bug report to Julia.

**Comment by arun3688** [comment]
I got the following measurements

```
arupa54@win00085 MINGW64 /c/OPENMODELICAGIT/OpenModelica/OMJulia.jl/src (master)
$ time julia lexertest.jl
Any[2]
  0.000088 seconds (21 allocations: 130.945 KiB)

real    0m3.958s
user    0m0.000s
sys     0m0.015s

arupa54@win00085 MINGW64 /c/OPENMODELICAGIT/OpenModelica/OMJulia.jl/src (master)
$ time julia lexertest.jl
Any[2]
  0.000092 seconds (21 allocations: 130.945 KiB)

real    0m3.851s
user    0m0.015s
sys     0m0.000s

arupa54@win00085 MINGW64 /c/OPENMODELICAGIT/OpenModelica/OMJulia.jl/src (master)
$ time julia lexertest.jl
Any[2]
  0.000125 seconds (21 allocations: 130.945 KiB)

real    0m4.055s
user    0m0.000s
sys     0m0.015s

```

**Comment by sjoelund** [comment]
From Slack

```
arupa54@win00085 MINGW64 /c/OPENMODELICAGIT/OpenModelica/OMJulia.jl/src (master)
$ time julia -O0 lexertest.jl
Any[2]
  0.000160 seconds (21 allocations: 130.945 KiB)
real    0m1.093s
user    0m0.015s
sys     0m0.015s
arupa54@win00085 MINGW64 /c/OPENMODELICAGIT/OpenModelica/OMJulia.jl/src (master)
$ time julia -O1 lexertest.jl
Any[2]
  0.000114 seconds (21 allocations: 130.945 KiB)
real    0m3.868s
user    0m0.000s
sys     0m0.015s
arupa54@win00085 MINGW64 /c/OPENMODELICAGIT/OpenModelica/OMJulia.jl/src (master)
$ time julia lexertest.jl
Any[2]
  0.000145 seconds (21 allocations: 130.945 KiB)
real    0m3.976s
user    0m0.015s
sys     0m0.015s
```

**Comment by sjoelund** [comment]
Should be fixed by #38
